test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -312,6 +331,12 @@ public void ParseValue ()
             Assert.Throws<ParseException>(() => Parsers.Value.End().Parse("0\n1"));
             Assert.Throws<ParseException>(() => Parsers.Value.End().Parse("'"));
 
+            Assert.Equal("0", Parsers.Value.Parse("0\n1").Value);   // value ends on linebreak
+            
+            // throw on unmatched quotes
+            Assert.Throws<ParseException>(() => Parsers.Value.Parse("'"));
+            Assert.Throws<ParseException>(() => Parsers.Value.Parse("\""));
```

Comment:
I believe I added a few more here -- were they duplicates? Or do you think they are not needed?

Reply:
Comments at the discussion below, closing this